File: JwtExample3/JwtExample3/src/main/java/com/jwt/example/service/QuestionService.java
package com.jwt.example.service;

import java.util.Set;

import com.jwt.example.entity.exam.Question;
import com.jwt.example.entity.exam.Quiz;

public interface QuestionService {
	
	public Question addQuestion(Question ques);
	public Question updateQuestion(Question ques);
	public Set<Question> getAllQuestions();
	public Question getQuestion(Long quesId);
	public void deleteQuestion(Long quesId);
	public Set<Question> getQuestionsOfQuiz(Quiz quiz);

}

File: examfront/src/services/question.service.ts
import { HttpClient } from '@angular/common/http';
import { Injectable } from '@angular/core';
import baseUrl from './helper';

@Injectable({
  providedIn: 'root'
})
export class QuestionService {

  constructor( private http : HttpClient) { }

  public getQuestionsOfQuiz(qId  : any){
    return this.http.get(`${baseUrl}/question/quiz/all/${qId}`);
  }
  public getQuestionByQuesId(quesId : any){
    return this.http.get(`${baseUrl}/question/${quesId}`);
  }

  public addQuestion(question : any, quiz : any)
  {
    const requestBody = {
      quiz: quiz,
      question: question,
    };
    console.log("rendering")
    console.log("question", question)
    return this.http.post(`${baseUrl}/question/add-question`,requestBody);
  }
  public deleteQuestion(que : any){
    return this.http.delete(`${baseUrl}/question/${que}`);
  }

  public updateQuestion(ques : any){
    return this.http.put(`${baseUrl}/question/update-question`,ques);
  }
}
